# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../openvas-server/install-includes.patch
# Copyright (C) 2007 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- openvas-server-0.9.0/Makefile.orig	2007-09-18 02:16:13.000000000 +0200
+++ openvas-server-0.9.0/Makefile	2007-09-18 02:18:20.000000000 +0200
@@ -79,7 +79,7 @@
 	$(INSTALL) -m 755 openvas-rmuser ${sbindir}/openvas-rmuser
 	$(INSTALL) -m 755 openvas-mkcert ${sbindir}/openvas-mkcert
 	$(INSTALL) -c -m 0444 openvas-services ${localstatedir}/lib/openvas/openvas-services
-	$(INSTALL) -c -m 0444 include/includes.h ${includedir}/openvas/includes.h
+	@test -f ${includedir}/openvas/includes.h || $(INSTALL) -c -m 0444 include/includes.h ${includedir}/openvas/includes.h
 	$(INSTALL) -c -m 0444 include/openvas-devel.h ${includedir}/openvas/openvas-devel.h
 	$(INSTALL) -c -m 0444 include/config.h ${includedir}/openvas/config.h
 	$(INSTALL) -c -m 0444 include/threadcompat.h ${includedir}/openvas/threadcompat.h
